NedyUdombat/ReachHigh
# Seed goals and task data

#### What does this PR do?
Seed goals and task data

#### Description of Task to be completed?
- [x] Seed goals data
- [x] Seed tasks data

#### How should this be manually tested?
* clone https://github.com/andela/zinnia-ah-backend.git
* checkout to this branch `ft-seed-data-169183130`
* pull from the remote branch to update the local branch
* run `npm install`
* run `cp .env.example .env`
* create a database
* fill in the details in your .env
* run `sequelize db:migrate`
* run `sequelize db:seed:all`
* open your database and you will see yur seeded data

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?(if applicable)
[Pivotal tracker ID](https://www.pivotaltracker.com/story/show/169183130)

#### Screenshots
